account_invoice_customer_no_autofollow/models/res_config_settings.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    no_auto_follow = fields.Boolean(
```

Comment:
field name should be more explicit
eg "invoice_partner_no_autofollow"

Reply:
Got it, I'll do it